packages/frontend/app/components/app/sidebar/chats/index.hbs

```diff
@@ -1,4 +1,4 @@
-<nav class='contacts' style='height: 100%; overflow-y: hidden;' data-test-contacts-list>
```

Comment:
why'd this change?

Reply:
There was an issue that was causing the tabs to pop up inside a scroll bar with the rest of the sidebar. This led to moving the tabs outside of the div that contained the list of chats/contacts/actions. The 'height: 100%' style was causing some issues with vertical alignment, and I found that removing it and adding a grid template line in the related .css file resolved the issues.